__init__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2017 Shotgun Software Inc.
```

Comment:
I don't think we should take this approach. This will make the repo a python package which I think will end up biting us if added to the python path (in terms of namespace collisions).